proxy.js

```diff
@@ -14,6 +13,9 @@ function reset() {
 }
 
 module.exports = {
+  setSaveDir(saveDir) {
+    pacScriptPath = path.join(saveDir, "proxy-pac.js");
+  },
```

Comment:
@ahmedalsudani agreed, although the intention here is not saving state, but being able to get information that is no longer accessible (saveDir contains the installerDir, which is now part of the launcher and not of common).